docs/source/torch.rst

```diff
@@ -553,6 +553,7 @@ Utilities
     result_type
     can_cast
     promote_types
+    use_deterministic_algorithms
     set_deterministic
```

Comment:
I think we can remove the doc entry for the deprecated function so people don't accidentally start using it